client/api/dto/dto.go

```diff
@@ -51,7 +51,7 @@ type StartDkgDTO struct {
 }
 
 type ProposeSignDataDTO struct {
-	DkgID string
+	DkgID []byte
```

Comment:
Maybe we should name them all in a uniform way? `DKGIdentifier` throughout the code